website/docs/r/organizations_account.html.markdown

```diff
@@ -12,7 +12,8 @@ Provides a resource to create a member account in the current organization.
 
 ~> **Note:** Account management must be done from the organization's master account.
 
-!> **WARNING:** Deleting this Terraform resource will only remove an AWS account from an organization. Terraform will not close the account. The member account must be prepared to be a standalone account beforehand. See the [AWS Organizations documentation](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_remove.html) for more information.
+!> **WARNING:** By default, deleting this Terraform resource will only remove an AWS account from an organization. To close the account you must set the `close_on_deletion` flag to true.
```

Comment:
Can you include a note about the account delete quota limits as well?

Reply:
Wording might be conversation driving on this, so placing as a suggestion to see what you think.
```suggestion
!> **WARNING:** By default, deleting this Terraform resource will only remove an AWS account from an organization. To close the account you must set the `close_on_deletion` flag to true. Please be aware that AWS enforces [a quota](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CloseAccount.html) on the number of accounts you close.
```

Reply:
Perhaps a separate note instead. What do you think?

```
~> **Note:** Quotas are enforced when using the [close_on_deletion argument](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CloseAccount.html). You can only close 10% of active member accounts within a rolling 30 day period. This quota is not bound by a calendar month, but starts when you close an account. Within 30 days of that initial account closure, you can't exceed the 10% account closure limit.
```

Thank you!

Reply:
I love your addition. My only concern is that if they change the percentage or the number of days the doc will fall out of place. Do you think it's worth mentioning the concrete numbers in that or generalizing. 
```
~> **Note:** Quotas are enforced when using the [close_on_deletion argument](https://docs.aws.amazon.com/organizations/latest/APIReference/API_CloseAccount.html). You can only close a percentage of active member accounts within a rolling 
period. Exceeding this quota, will lead to errors and require you to manually close the accounts. 
```

Reply:
@drewmullen @danquack Could you please come to a resolution on this soon?
I am reviewing this PR at the moment and I have no other comments on the documentation.
Thanks.

Reply:
Talked to drew and we settled on referencing the error with linkage to the documentation, vs calling out the specific quota of the error.